spinalcordtoolbox/download.py

```diff
@@ -139,6 +140,27 @@
         "default_location": os.path.join(__sct_dir__, "data", "PAM50_normalized_metrics"),
         "download_type": "Templates",
     },
+    "softseg_models": {
+        "mirrors": [
+            "https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/releases/download/v2.0/model_2023-09-18.zip"
+        ],
+        "default_location": os.path.join(__sct_dir__, "data", "softseg_models"),
+        "download_type": "Models",
+    },
+    "sci_models": {
+        "mirrors": [
+            "https://github.com/joshuacwnewton/model_seg_sci/releases/download/r20230814/model-sci-multisite_r20230814.zip",
+        ],
+        "default_location": os.path.join(__sct_dir__, "data", "sci_models"),
+        "download_type": "Models",
+    },
+    "rootlet_models": {
+        "mirrors": [
+            "https://github.com/joshuacwnewton/model-spinal-rootlets/releases/download/r20230815/model-spinal-rootlets_r20230815.zip"
```

Comment:
Last week, I finished training of a new (better) version of the rootlets model (internally called [model M5 fold_all](https://github.com/ivadomed/model-spinal-rootlets/pull/23#issuecomment-1888047165)). Can I share the model with you to update this link?

Reply:
Of course!! :) 

Though, the model itself shouldn't matter too much right this second, as I'm just working out the actual interface, rather than any specific results. If you want, feel free to wait until there's a proper release on the project repo itself!